src/java/org/apache/cassandra/service/accord/serializers/DepsSerializer.java

```diff
@@ -46,93 +46,133 @@
-        }
+    public static DepsSerializer<Deps> deps()
+    {
+        return instance.deps;
+    }
 
-        @Override
-        public long serializedSize(PartialDeps partialDeps, int version)
-        {
-            return super.serializedSize(partialDeps, version)
-                 + KeySerializers.participants.serializedSize(partialDeps.covering, version);
-        }
+    public static IVersionedSerializer<Deps> nullableDeps()
+    {
+        return instance.deps;
+    }
 
-        @Override
-        public long serializedSize(Unseekables<?> keys, PartialDeps partialDeps, int version)
-        {
-            return super.serializedSize(keys, partialDeps, version)
-                 + KeySerializers.participants.serializedSize(partialDeps.covering, version);
-        }
-    };
+    public static DepsSerializer<PartialDeps> partialDeps()
+    {
+        return instance.partialDeps;
+    }
 
-    public static final IVersionedSerializer<PartialDeps> nullablePartialDeps = NullableSerializer.wrap(partialDeps);
+    public static IVersionedSerializer<PartialDeps> nullablePartialDeps()
+    {
+        return instance.nullablePartialDeps;
+    }
 
     abstract D deserialize(KeyDeps keyDeps, RangeDeps rangeDeps, KeyDeps directKeyDeps, DataInputPlus in, int version) throws IOException;
 
     @Override
     public void serialize(D deps, DataOutputPlus out, int version) throws IOException
     {
-        KeySerializers.routingKeys.serialize(deps.keyDeps.keys(), out, version);
+        KeySerializers.routingKeys().serialize(deps.keyDeps.keys(), out, version);
         serializeWithoutKeys(deps, out, version);
     }
 
     @Override
     public void serialize(Unseekables<?> superset, D deps, DataOutputPlus out, int version) throws IOException
     {
         if (superset.domain() == Key) serializeSubset(deps.keyDeps.keys(), superset, out);
-        else KeySerializers.routingKeys.serialize(deps.keyDeps.keys(), out, version);
+        else KeySerializers.routingKeys().serialize(deps.keyDeps.keys(), out, version);
         serializeWithoutKeys(deps, out, version);
     }
 
     @Override
     public D deserialize(DataInputPlus in, int version) throws IOException
     {
-        RoutingKeys keys = KeySerializers.routingKeys.deserialize(in, version);
+        RoutingKeys keys = KeySerializers.routingKeys().deserialize(in, version);
         return deserializeWithoutKeys(keys, in, version);
     }
 
     @Override
     public D deserialize(Unseekables<?> superset, DataInputPlus in, int version) throws IOException
     {
         RoutingKeys keys;
-        if (superset.domain() == Key) keys = ((AbstractUnseekableKeys)deserializeSubset(superset, in)).toParticipants();
-        else keys = KeySerializers.routingKeys.deserialize(in, version);
+        if (superset.domain() == Key)
+            keys = ((AbstractUnseekableKeys) deserializeSubset(superset, in)).toParticipants();
```

Comment:
either new line and indent both or neither

Reply:
this was auto-formatting. Fixed to neither